crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/main.py

```
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)